src/components/shared/stateNew/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { count } from 'console'
```

Comment:
what did you write for? what is he doing?

Reply:
console-ic(module) importa anum county